### ISSUE TITLE
Create videos about the major changes in Backdrop

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#130]
Post these on the blog, but link from the home page where each item is described.

**Comment by klonos** [comment]
These can/should be hosted on YouTube I think.

**Comment by jenlampton** [comment]
yes yes. And best yet - they can be done by anyone! :) volunteers welcome.

**Comment by laryn** [comment]
I am going to try to make one or two of these and see how they go. If anyone is willing to jot some of the key points they see in various categories so important things don't get left out, feel free to edit here:

https://docs.google.com/document/d/1RCq8QIOEOaegCJkmY5yLlJmIaIfvUdu3cd1wnKjo4Og/edit?usp=sharing

**Comment by jenlampton** [comment]
That would be fantastic! :)

**Comment by stpaultim** [comment]
I'm updating this issue a bit to add some information for a potential screencast volunteer that expressed interest on Twitter today. Please, add your ideas for videos/screencasts that would be helpful.
